### PR TITLE
Add size

### PR DESCRIPTION
**Pull request by regevbr**
add a length property #14

### CODE REVIEW

**Comment by regevbr** [comment]
@wooorm can you please code review?

**Comment by regevbr** [comment]
@wooorm I thought I will maintain the list convention but you are correct. Maybe we can use `size` instead? Like used in [Set](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Set/size)?

**Comment by wooorm** [comment]
Oh `size` like maps and sets is good! Yeah!

**Comment by regevbr** [comment]
Ok I will get right on it!

File: readme.md [hunk 1]
```diff
@@ -189,6 +189,24 @@ list.tail === item2 // => true
...
+list.length === 0 // => true
+
+list.append(item1)
+list.length === 1 // => true
+
+list.append(item2)
+list.length === 2 // => true
+```
+
+The length of the list.
```
Comment (wooorm) [inline readme.md]:
```suggestion
The number of items in the list.
```

**Comment by regevbr** [comment]
@wooorm can you please re-review?

**Pull request merged**